src/sdk/namespace_s3.js

```diff
@@ -95,9 +95,19 @@ class NamespaceS3 {
             common_prefixes: _.map(res.CommonPrefixes, 'Prefix'),
             is_truncated: res.IsTruncated,
             next_marker: res.NextMarker,
+            object_owner: await this.get_object_owner(object_sdk, params.bucket),
         };
     }
 
+    async get_object_owner(object_sdk, bucket) {
+        // TODO: in the future we will add extra implementation per namespace
+        const info = await object_sdk.get_bucket_owner(bucket);
+        return {
+              name: info.bucket_owner.unwrap(),
+              id: info.bucket_info.owner_account.id,
+          };
+    }
+
```

Comment:
```suggestion
    async get_object_owner(object_sdk, bucket) {
        // TODO: in the future we will add extra implementation per namespace
        return object_sdk.get_bucket_owner(bucket);
    }

```

Reply:
please do that on all namespace

Reply:
please also add JSdoc on all new functions

Reply:
Sure. I will add the JSdoc.

code changes depend on https://github.com/noobaa/noobaa-core/pull/8062#discussion_r1673766705